appveyor.yml

```diff
@@ -0,0 +1,27 @@
+version: 1.0.0.{build}
+image:
+  - Visual Studio 2017
+pull_requests:
+  do_not_increment_build_number: true
+branches:
+  only:
+    - master
+skip_branch_with_pr: true
+configuration: Release
+before_build:
+  - cmd: dotnet restore
+build:
+  verbosity: minimal
+  parallel: true
+build_script:
+  - cmd: dotnet build -c Release
+test_script:
+  - cmd: dotnet test -c Release
+artifacts:
+  - path: '**\*.nupkg'
+nuget:
+  disable_publish_on_pr: true
+#deploy:
+#  provider: NuGet
+#  api_key:
+#    secure: <Your secret>
```

Comment:
@JohannesMoersch CD can be achieved by setting up api key in this deploy section

Reply:
I don't want to put the API key in a public repo though. I'm guessing there's a way to configure that to be injected from the configuration in the web interface though. I'll look into it.

Reply:
@JohannesMoersch  The api_key you put here is encoded, no one else can use it, check this out: API key (api_key) - your API key. Use “Encrypt configuration data” page in AppVeyor (Settings → Encrypt YAML) to encrypt your API key before saving it to appveyor.yml.

Reply:
Oh, okay, that makes sense. Thanks. Any thoughts on how to trigger CI on an update to a release notes file? I don't want to publish on every PR merge. Ideally, I'd have to add a new line to a release notes file with a version number, and it would trigger a deploy using that version number.

Reply:
@JohannesMoersch It will skip publishing when package version is not changed, maybe consider only changing version number on publish? Or mark unstable versions with beta / dev suffix, like many MS official packages do

Reply:
@JohannesMoersch  Or maybe create a dedicate publish branch : )

Reply:
I didn't realize it only published on version changes. Okay, cool. Thanks

Reply:
It still tries to publish but will skip when the exact version exists on nuget